src/memory/memory.c

```diff
@@ -810,3 +832,42 @@ bool memory_multisig_get_by_hash(const uint8_t* hash, char* name_out)
     }
     return false;
 }
+
+static int _ctap_resident_key_chunk(int key_idx)
+{
+    return CHUNK_3 + key_idx / MEMORY_CTAP_RESIDENT_KEYS_PER_CHUNK;
```

Comment:
What if `key_idx` is `46`? Then I suppose `_read_chunk` below tries reading from an invalid chunk 5. Haven't check but can `_read_chunk` handle that?